package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "service-template-node",
-  "version": "0.0.0",
+  "version": "0.1.0",
```

Comment:
IMHO it is too premature to be setting the version to `0.1.0`. I'd set it to `0.0.1` or perhaps `0.0.2`, not more than that. Especially since we plan to start using swagger route configuration and generation, which would warrant a change to `0.1.0`